src/multi-level-checkbox-editor/node-box.tsx

```diff
@@ -11,6 +11,7 @@ interface Props {
     onNodeClick: (node: Node) => void;
     setNodeValue: (node: Node, value: boolean) => void;
     levelHeaderLabel: string;
+    checkBoxHeader?: boolean;
```

Comment:
What about renaming this to `isCheckBoxHeaderDisabled`,`isCheckBoxHeaderVisable` or`isCheckBoxHeader` or any way that makes it questionable?

Reply:
[Deprecated comment] I didn't notice Benjamin has already commented the same,
https://github.com/Webtrekk/Ampel-UI/pull/58#discussion_r389734890

Reply:
no problem